cluster-operator/src/main/java/io/strimzi/operator/cluster/model/AbstractModel.java

```diff
@@ -514,6 +514,14 @@ public static String getPersistentVolumeClaimName(String name, int podId) {
         return VOLUME_NAME + "-" + name + "-" + podId;
     }
 
+    public static String getPersistentVolumeClaimName(StatefulSet ss) {
```

Comment:
This method seems to be new in this PR but doesn't seem to be used anywhere ... is this some preparation for the future? Or some leftover which should be removed?